src/test/groovy/com/palantir/gradle/versions/CheckOverbroadConstraintsIntegrationSpec.groovy

```diff
@@ -0,0 +1,135 @@
+/*
+ * (c) Copyright 2019 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.gradle.versions
+
+import org.gradle.testkit.runner.BuildResult
+import org.gradle.testkit.runner.TaskOutcome
+
+class CheckOverbroadConstraintsIntegrationSpec extends IntegrationSpec {
+
+    def setup() {
+        buildFile << """
+            plugins {
+                id 'com.palantir.versions-lock'
+                id 'com.palantir.versions-props'
+            }
+        """.stripIndent(true)
+        createFile('versions.props')
+        createFile('versions.lock')
+
+        createFile('gradle.properties') << """
+            gcv.overbroad.constraints=true
+        """.stripIndent(true)
+    }
+
+    def buildSucceed() {
+        BuildResult result = runTasks('checkOverbroadConstraints')
```

Comment:
You can just use `runTasksSuccessfully` here.

Reply:
Oh wait, is this the testkit runner?

Reply:
I'm not sure that the testkit runner is but this seem to be how all the groovy tests are done in gcv